docs/guides/installing-unreleased-changes.md

```diff
@@ -6,8 +6,8 @@ How to install local Carbon into your application
 
 1. Pull down the Carbon repository locally.
 2. Checkout the appropriate branch which contains the changes you want to install.
-3. Run `npm run-script prepublishOnly` - This will compile the code from ES6/ES7 into ES5 for the browser to process.
+3. Run `npm run precompile` - This will compile the code from ES6/ES7 into ES5 for the browser to process.
 4. Within your local application that uses Carbon. Change the package.json file so that carbon points to a relative path `./../carbon`.
-5. Run `npm install carbon-react` or `npm install` to install the local version of the application.
+5. Run `npm install $(npm pack ./../../carbon/ | tail -1)` to install the local version of the application.
```

Comment:
We don't need step 4 anymore. Probably also worth making a note that the relative path may be different depending where you're installing it from.